src/components/monitoringTable/MonitoringTable.tsx

```diff
@@ -119,6 +118,20 @@ export const MonitoringTable: React.FC = () => {
     onError: (error) => updateSnackbar(true, t('errorRequest', { ns: 'general' }), 'error'),
   })
 
+  const generatePath = (eserviceRecordId: string) => {
+    let path = `/monitoraggio/eservice/${eserviceRecordId}`
+    if (sessionStorage.getItem('token')) {
+      const startDate = format(subDays(new Date(), 2), "yyyy-MM-dd'T'HH:mm:ss.sss'Z'").replaceAll(
+        ':',
+        '%3A'
+      )
+      const endDate = format(new Date(), "yyyy-MM-dd'T'HH:mm:ss.sss'Z'").replaceAll(':', '%3A')
+      path = `${path}?startDate=${startDate}&endDate=${endDate}`
```

Comment:
Penso qui sia più comodo l'utilizzo di `URLSearchParams` che con il metodo `toString()` genera i parametri url con la codifica corretta.

```suggestion
const urlParams = new URLSearchParams({
  startDate: format(subDays(new Date(), 2), "yyyy-MM-dd'T'HH:mm:ss.sss'Z'"),
  endDate: format(new Date(), "yyyy-MM-dd'T'HH:mm:ss.sss'Z'"),
}).toString()
path = `${path}?${urlParams}````